project-base/app/elFinderVolumeFlysystem.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+use Shopsys\ShopBundle\Component\Flysystem\VolumeDriver;
+
+class elFinderVolumeFlysystem extends VolumeDriver {}
```

Comment:
again, why?

Reply:
Why is it not possible to just extend classses from framework package and overwrite methods? why does it need to be registered as classmap? 

I would maybe have these files inside of framework since i dont see any reason why this kind of stuff should somoene have in a project-base.

Reply:
> Even more this file violates PSR-4 standards

My mistake sorry

Reply:
i am for any better solution